Assets/MixedRealityToolkit/InputModule/Scripts/Utilities/Interactions/TwoHandManipulatable.cs

```diff
@@ -0,0 +1,631 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+// #defines for features that are not yet implemented
+#define TODO_ROTATE_FACE_USER
```

Comment:
What's this doing exactly?

I'm not sure we should be defining our own symbols.

Reply:
It's just a way for me to keep track of TO DO items. I removed it.